py/client/pydeephaven/session.py

```diff
@@ -100,8 +103,20 @@ def __init__(self, host: str = None, port: int = None, auth_type: str = "Anonymo
             auth_token (str): the authentication token string. When auth_type is 'Basic', it must be
                 "user:password"; when auth_type is "Anonymous', it will be ignored; when auth_type is a custom-built
                 authenticator, it must conform to the specific requirement of the authenticator
-            never_timeout (bool, optional): never allow the session to timeout, default is True
-            session_type (str, optional): the Deephaven session type. Defaults to 'python'
+            never_timeout (bool): never allow the session to timeout, default is True
+            session_type (str): the Deephaven session type. Defaults to 'python'
+            use_tls (bool): if True, use a TLS connection.  Defaults to False
+            pem (bytes): PEM encoded certificate to use for TLS connection, or None to use system defaults.
+                 If not None implies use a TLS connection and the use_tls argument should have been passed
+                 as True. Defaults to None
```

Comment:
is this the root cert? or the certs of the client?

Reply:
root, I'll make the doc more specific